Formula/firefoxpwa.rb

```diff
@@ -0,0 +1,61 @@
+class Firefoxpwa < Formula
+  desc "Tool to install, manage and use Progressive Web Apps in Mozilla Firefox"
+  homepage "https://github.com/filips123/FirefoxPWA"
+  url "https://github.com/filips123/FirefoxPWA/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "47cb03a8f5773da235e360655a4ef93203d7d3ed760ebf6013692ec20239c2c1"
+  license "MPL-2.0"
+  head "https://github.com/filips123/FirefoxPWA.git"
+
+  depends_on "rust" => :build
+
+  def install
+    cd "native"
+
+    # Prepare the project to work with Homebrew
+    ENV["FFPWA_EXECUTABLES"] = bin
+    ENV["FFPWA_SYSDATA"] = share
+    system "bash", "./packages/brew/configure.sh", version, bin, libexec
+
+    # Build and install the project
+    system "cargo", "install", *std_cargo_args
+
+    # Install all files
+    libexec.install bin/"firefoxpwa-connector"
+    share.install "manifests/brew.json" => "firefoxpwa.json"
+    share.install "userchrome/"
+    bash_completion.install "target/release/completions/firefoxpwa.bash" => "firefoxpwa"
+    fish_completion.install "target/release/completions/firefoxpwa.fish"
+    zsh_completion.install "target/release/completions/_firefoxpwa"
+  end
+
+  def caveats
+    filename = "firefoxpwa.json"
+
+    source = share
```

Comment:
```suggestion
    source = opt_share
```

Reply:
What's the difference between `share` and `opt_share`? It is not mentioned in Formula Cookbook.

Reply:
`opt_` variables use a version independent path. For example `/usr/local/opt/php` vs `/usr/local/Cellar/8.0.0/php` because the latter will disappear when 8.0.1 comes out.

Reply:
If you use `share`, then a user's symlink will break as soon as `firefoxpwa`'s version or revision is updated. `opt_share` will give you a path that that doesn't depend on the formula version (or the internal `pkgversion` we use whenever we want to ship an update to users without any upstream changes).

Reply:
Thanks for explanation!